vuepress/docs/next/tutorials/ecr/github-actions-workflow.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebarDepth: 2
+---
+
+# Add a GitHub Actions CI Workflow
+
+## Overview
+This tutorial shows how to use the JHipster `ci-cd` sub-generator to quickly create a basic GitHub Actions Workflow for the microservices and micro frontends generated by the Entando Component Generator.
+
+## Prerequisites
+* An existing project with a GitHub repository. See the [Entando Component Generator](../backend-developers/generate-microservices-and-micro-frontends.md#overview) to create a new one.
+* [GitHub Actions](https://github.com/features/actions). These are enabled by default without additional configuration. Please note public repositories currently qualify for unlimited usage but private repositories can encounter usage restrictions.
+* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) to verify you have the command line prerequisites in place for this tutorial (e.g. npm, git, JHipster).
+
+## Create a backend workflow
+
+1. Go to your main project folder in the shell
+2. Run the JHipster `ci-cd` subgenerator
+```bash
+ent jhipster ci-cd
+```   
+2. Select `GitHub Actions`
+3. Click ENTER to choose no `tasks/integrations` and generate the default workflow
+4. The initial workflow file is available at `.github/workflows/github-ci.yml`
+5. Commit the workflow file and push it to github.
+```bash
+git add .github
+git commit -m "Add the backend CI job"
+git push
+```
+6. Since the workflow runs on ```push``` or ```pull-request```, you can immediately check the GitHub ```Repository → Actions``` tab to see the status of the corresponding Jobs. The standard tests in a Blueprint-generated project can take a few minutes to run.
+* You can review the logs for each step to diagnose CI failures
+* If the workflow fails or times out, you should receive an email notification
+* You can choose to skip the CI workflow by including `skip ci` or `ci skip` in your commit message. This can be customized in the job definition.
```

Comment:
same as above wrt Job or job...

Reply:
Same, lowercase job, I think. It's lowercase in the yaml so probably good to match it here.